kumascript/src/info.js

```diff
@@ -147,6 +150,17 @@ const info = {
       }
       return {};
     }
+    return this.getPage(document);
+  },
+
+  getPage(document) {
+    if (typeof document === "string") {
```

Comment:
What do you think? I don't think it ever happens anymore with our en-us content. So I could replace this with a throw instead. And if so, I'd perhaps do something like this (pseudo code...):
```javascript
if (!document.url || !document.metadata) {
  // Code comment here that explains why we're doing this. 
  throw new Error("getPage() was called with a document that doesn't have 'url' or 'metadata' (${JSON.stringify(document)})");
}
```
Thing is, I'm still not sure I trust KS to not swallow a throw there and then turn them into flaws that we don't notice properly.

Reply:
> Thing is, I'm still not sure I trust KS to not swallow a throw there and then turn them into flaws that we don't notice properly.

At least as a start, I like what you have here for this very reason.